new: Historical Maps

sppmacd opened the pull request:
This is a page which would display Prosperity maps of various highly active regions, downloaded every day, so that you can look how the server changed with time. Then after a year or something we could do a timelapse 🙂

Note that:
* This requires some backend server which would download and host the images.
* **This is not ready to merge**. I just hardcoded localhost as that server.
* I'm open to suggestions for regions that will be downloaded and displayed in full resolution, as downloading the whole map (over 40000x40000 per dimension) is way too much for now

mrmelon54 commented:
Do you need someone to make the API?

sppmacd commented:
> Do you need someone to make the API?

I have [written](https://github.com/sppmacd/Prosperity-Map-Server) all the required scripts, just need someone who will host them and run fetch every day.

CarbonGhost commented:
Any chance this could utilize the free resources provided with our Vercel plan? I would really like this to be as simple as possible to maintain

mrmelon54 commented:
I'm sure we can figure something out...

Not sure how yet?

sppmacd commented:
In the worst case, I can host the image storage and the website would just fetch from there.

sppmacd commented:
I implemented daily fetch on my map server repo: https://github.com/sppmacd/Prosperity-Map-Server. Also updated the website so that it fetches from there :)

sppmacd commented:
New repo with tiled maps: https://github.com/sppmacd/Prosperity-Maps
